src/main/java/org/opensearch/jdbc/ConnectionImpl.java

```diff
@@ -507,7 +507,7 @@ public Logger getLog() {
     }
 
     private String getUserAgent() {
-        return String.format("openes-jdbc/%s (Java %s)",
+        return String.format("opensearch-jdbc/%s (Java %s)",
```

Comment:
Are there any clients that rely on string value equality?

Reply:
good question  - this originally looks like a type err - should I keep it for the case you mentioned ?

Reply:
I'll revert this just to be on the same side